java/scene-property-expressions/README.md

```diff
@@ -2,53 +2,35 @@
 
 Update the orientation of a graphic using expressions based on its attributes.
 
-![Scene property expressions App](scene-property-expressions.png)
+![Image of scene property expressions](scene-property-expressions.png)
 
 ## Use case
 
-Instead of reading the attribute and changing the rotation on the symbol
-for a single graphic (a manual CPU operation), you can bind the rotation
-to an expression that applies to the whole overlay (an automatic GPU
-operation). This usually results in a noticeable performance boost
-(smooth rotations).
+Instead of reading the attribute and changing the rotation on the symbol for a single graphic (a manual CPU operation), you can bind the rotation to an expression that applies to the whole overlay (an automatic GPU operation). This usually results in a noticeable performance boost (smooth rotations).
 
 ## How to use the sample
 
-Use the seek bars to adjust the heading and pitch properties for the
-graphic.
+Adjust the heading and pitch sliders to rotate the cone.
 
 ## How it works
 
-Simple renderers can dynamically update the positions of graphics using
-an expression. The expression relates a renderer property to one of the
-graphic's attribute keys.
-
 1. Create a new `GraphicsOverlay`. 
 2. Create a new `SimpleRenderer` and set its scene properties.
-3. Set the heading expression to `[HEADING]` and the pitch expression to
-   `[PITCH]`.
-4. Apply the renderer to the graphics overlay.
-5. Create a new `Point` and a new `Graphic` and add it to the overlay.
-6. To update the graphic's rotation, update the `HEADING` or `PITCH`
-   property in the graphic's attributes.
+3. Set the heading expression to `[HEADING]` and the pitch expression to `[PITCH]` with `renderer.getSceneProperties().setHeadingExpression`.
+4. Apply the renderer to the graphics overlay with `graphicsOverlay.setRenderer(renderer)`.
+5. Create a new `Point` and a new `Graphic` and add it to the overlay with e.g. `graphicsOverlay.getGraphics().add(graphic)`.
+6. To update the graphic's rotation, update the `HEADING` or `PITCH` property in the graphic's attributes with `graphic.getAttributes().put(key, value)`.
        
 ## Relevant API
 
+* Graphic.Attributes
```

Comment:
Is this also meant to be just `Graphic`?

Reply:
Same for the ones below ⬇️

Reply:
ah, thanks! @sorenoid and I had a wee chat about use of methods in the Relevant API section at the end of the day yesterday. The summary of that discussion was that it turns out it is ok to refer to methods in this section, but that the Java/Android API will be more likely just to refer to classes in this section (it's a different story for other SDKs). 

However, we agreed if there was a particular method that we thought was deserving of being left in or directly relevant to the sample we could leave it in. This is where yourself and Soren's new exposure to our API will come in handy to judge if we should leave some methods in, if you think it adds to the readme. I think this one is a good example. Here, I'll remove Graphic.getAttribute since it's not pertinent to the sample, however I'll leave in SceneProperties.setHeading/PitchExpression since these are key for this sample. 

Going forward though I wouldn't add any methods to designs which don't already refer to them: and if they do, let's use our own judgement from a Java/Android perspective to assess if they add to the readme or not.

Reply:
Oh no I saw this after going through one of Soren's PRs and it corrected all these 🙈 I'll keep this in mind.